libs/domains/organizations/feature/src/lib/git-branch-settings/git-branch-settings.tsx

```diff
@@ -76,24 +77,26 @@ export function GitBranchSettings({ disabled, gitProvider }: GitBranchSettingsPr
           />
         )}
       />
-      <Controller
-        name="root_path"
-        control={control}
-        defaultValue="/"
-        rules={{
-          required: 'Value required',
-        }}
-        render={({ field, fieldState: { error } }) => (
-          <InputText
-            label="Root application path"
-            name={field.name}
-            onChange={field.onChange}
-            value={field.value}
-            error={error?.message}
-            disabled={disabled}
-          />
-        )}
-      />
+      {!hideRootPath && (
+        <Controller
```

Comment:
Shouldn't we have 2 dedicated components for the branch and the rootPath ?

Reply:
Yes, it should be cleaner, but I need to manage all services with git branch for it